cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
@psav Does num_image() check the number of images in the API of Openshift or CFME? Apparently, there is a new feature in CFME 5.7 that displays the number of images that is larger than the number of images in Openshift. For instance, when I check the Openshift, it shows me 87 images vs. 100 images in CFME. When I checked num_image() in iPython, I got 100 images as well. So there is a discrepancy here.